flutter-mentorship-t4/mentorship_project
# Feature (Sign Up) ✨Finishing most of Sign up logic

**Jira Ticket :** (https://nz-dev.atlassian.net/browse/DIVA-12)
**Description � :** 
  Complelte Coding of most Logic of Sign up Screen With connection With Firebase With Android &IOS

 **The following sections have been fully integrated:** 
          1- modifying SignUp Screen First 
          2- Running Name & email &Password Validation
          3- navigation to Login After Sign Up Success 
          4- Display a success message upon successful registration
          5- Show error messages if the email format is incorrect or the password does not meet the required strength.
**How to Test 📋:**
       **.** Open the app and navigate to the SignUp screen on various iPhone devices.
       **.** Verify that all elements are correctly positioned And Diong What it have to do.
       **.** Open the app and navigate to the SignUp screen on various iPhone devices.
       **.** Open the app and navigate to the SignUp screen on various iPhone devices.
**Screenshots/ Screen Recording 📸 :**
**Before:**
![Android Emulator - Pixel_5_API_29_5554 10_3_2024 12_59_53 AM](https://github.com/user-attachments/assets/b6f61c5a-9c57-45f5-9b32-90dc915e88d9)

**After:**
https://github.com/user-attachments/assets/d049140b-0edd-4940-b7d4-4fc6deaf906a
